debug/leaflet-include.js

```diff
@@ -37,7 +37,12 @@
 
 	var path = getSrcUrl();
     for (var i = 0; i < scripts.length; i++) {
-		document.writeln("<script src='" + path + scripts[i] + "'></script>");
+		if (scripts[i].substr(-3) === '.js') {
+			document.writeln("<script src='" + path + scripts[i] + "'></script>");
+		}
+		if (scripts[i].substr(-4) === '.css') {

```

Comment:
Feel free to add commits to this PR to tweak variable names and the like. It's not that important for me :-P